x-pack/plugins/embeddable_enhanced/public/actions/panel_notifications_action.test.ts

```diff
@@ -5,81 +5,113 @@
  * 2.0.
  */
 
-import { PanelNotificationsAction } from './panel_notifications_action';
-import { EnhancedEmbeddableContext } from '../types';
 import { ViewMode } from '@kbn/embeddable-plugin/public';
+import { EmbeddableApiContext, PublishesWritableViewMode } from '@kbn/presentation-publishing';
+import { DynamicActionManager } from '@kbn/ui-actions-enhanced-plugin/public/dynamic_actions';
+import { BehaviorSubject } from 'rxjs';
+import { HasDynamicActions } from '../embeddables/interfaces/has_dynamic_actions';
+import { DynamicActionsSerializedState } from '../plugin';
+import {
+  PanelNotificationsAction,
+  PanelNotificationsActionApi,
+} from './panel_notifications_action';
+
+const createContext = (events: any[] = []): EmbeddableApiContext => {
```

Comment:
how about changing `any[]` to `SerializedEvent[]`?

Reply:
Done in 343e464a979251cae24177f87eb348c52ab673b0